tensorboard/backend/application_test.py

```diff
@@ -543,27 +555,53 @@ def setUp(self):
               FakePlugin,
               plugin_name='bar',
               is_active_value=True,
-              routes_mapping={'/bar_route': self._bar_handler},
+              routes_mapping={
+                  '/bar_route': self._bar_handler,
+                  '/wildcard/*': self._wildcard_handler,
+                  '/wildcard/special/*': self._wildcard_special_handler,
+                  '/wildcard/special/exact': self._foo_handler,
+              },
               construction_callback=self._construction_callback)),
         ],
         dummy_assets_zip_provider)
 
-  def _foo_handler(self):
-    pass
-
-  def _bar_handler(self):
-    pass
+    self.server = werkzeug_test.Client(self.app, wrappers.BaseResponse)
 
   def _construction_callback(self, context):
     """Called when a plugin is constructed."""
     self.context = context
 
+  def _test_route(self, route, expected_status_code):
+    response = self.server.get(route)
+    self.assertEquals(response.status_code, expected_status_code)
```

Comment:
The `assertEquals` method is deprecated; prefer `assertEqual`.

Reply:
done